frontend_server/lib/server.dart

```diff
@@ -192,35 +184,28 @@ class _FrontendCompiler implements CompilerInterface {
   Future<Null> recompileDelta() async {
     final String boundaryKey = new Uuid().generateV4();
     _outputStream.writeln("result $boundaryKey");
-    final DeltaProgram deltaProgram = await _generator.computeDelta();
+    final Program deltaProgram = await _generator.computeDelta();
     final IOSink sink = new File(_kernelBinaryFilename).openWrite();
     final BinaryPrinter printer = printerFactory.newBinaryPrinter(sink);
-    printer.writeProgramFile(deltaProgram.newProgram);
+    printer.writeProgramFile(deltaProgram);
     await sink.close();
     _outputStream.writeln("$boundaryKey $_kernelBinaryFilename");
     return null;
   }
 
-  @override
-  void acceptLastDelta() {
-    _generator.acceptLastDelta();
-  }
-
-  @override
-  void rejectLastDelta() {
-    _generator.rejectLastDelta();
-  }
-
```

Comment:
I am not sure why these functions should be deleted and not just made nops temporarily with a TODO that refers to the issue that needs fixing. I have created a dart issue here https://github.com/dart-lang/sdk/issues/31919

Reply:
We will address this issue after this CL lands.